build/Dockerfile

```diff
@@ -151,6 +152,10 @@ RUN if [[ "${ENABLE_AIM}" == "true" ]]; then \
         python -m pip install --user "$(head bdist_name)[aim]"; \
     fi
 
+RUN if [[ "${ENABLE_MLFLOW}" == "true" ]]; then \
```

Comment:
Do we want to only install this package when both `ENABLE_AIM` and `ENABLE_MLFLOW` are set to true? What happens if `ENABLE_AIM` is false and `ENABLE_MLFLOW` is true, we would install here, which is not what we want right?

Reply:
I think I understand now, they are independent. We don't need AIM to enable MLFLOW

Reply:
yes we do not need AIM..they are independent packages.